contracts/TopShot.cdc

```diff
@@ -421,15 +422,15 @@ pub contract TopShot: NonFungibleToken {
         //
         // Returns: The NFT that was minted
         //
-        pub fun mintMomentWithSubedition(playID: UInt32, subeditionID: UInt32): @NFT {
+        access(all) fun mintMomentWithSubedition(playID: UInt32, subeditionID: UInt32): @NFT {
```

Comment:
`NFTMinter` entitlement?

Reply:
Is the entitlement needed in this case? The Set is a resource that's managed only by the `Admin` resource. Did you mean adding it as a way to further ensure any unauthorized operation?

Reply:
I don't understand it well enough, so I just wanted to flag in case it might be needed, but I don't think it is needed here for the reasons that you've described